options.go

```diff
@@ -31,17 +31,20 @@ type Options struct {
 	// Other options for implementations of the interface
 	// can be stored in a context
 	Context context.Context
+
+	InstallSignalHandler bool
```

Comment:
// Don't use internal os.Signal handler
NoSignal bool

Reply:
`Signal bool` which should default to true

Reply:
ah ok